src/Pages/Main/Component/AccomodationType/AccomodationType.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from 'react';
+import { API } from '../../../../config';
+import styled from 'styled-components';
+
+const AccomodationType = ({ accomoType, setAccomoType }) => {
+  const [categoryType, setCategoryType] = useState([]);
+  const [currentId, setCurrentId] = useState(0);
+
+  const handleColor = (e, name) => {
+    setCurrentId(e.target.id);
+    setAccomoType(name);
+  };
+
+  useEffect(() => {
+    fetch(`${API}/hotels/main`)
+      .then(res => res.json())
+      .then(res => {
+        setCategoryType(res.results.category);
+      });
+  }, []);
+
+  return (
+    <>
+      <Wrapper>
+        {categoryType.map((img, index) => (
+          <ImageBox>
+            <Image
+              id={index}
+              src={img.img_url}
+              alt={img.name}
+              onClick={e => handleColor(e, `${img.name}`)}
+            />
+            <Title color={setCurrentId === index && 'color'}>{img.name}</Title>
+          </ImageBox>
+        ))}
+      </Wrapper>
+    </>
+  );
+};
+
+export default AccomodationType;
+
+const Wrapper = styled.div`
+  ${props => props.theme.flexCenter};
+  height: 90px;
+`;
+
+const ImageBox = styled.div`
+  ${props => props.theme.FlexVertical}
+`;
+
+const Image = styled.img`
+  width: 50px;
+  height: 50px;
+  margin-right: 30px;
+  cursor: pointer;
+`;
+const Title = styled.span`
+  color: ${props =>
+    props.color ? `${props => props.theme.mainColor}` : '#4d4d4d'};
```

Comment:
```suggestion
  color: ${({ color, theme }) => color ? theme.mainColor : '#4d4d4d'};
```
- 남선님께서 공유해주신 내용 처럼 함수 인자 자리에서 직접 구조분해할당 해줄 수 있습니다
- props로 이미 꺼내어 오고 있는 값이 있는데 `${props => props.theme.mainColor}` 처럼 한번 더 꺼내 오는 건 어색해보입니다

Reply:
감사합니다 다른 것들도...수정하겠습니다!